src/pages/AddPersonalBankAccountPage.js

```diff
@@ -10,20 +10,27 @@ const propTypes = {
     ...withLocalizePropTypes,
 };
 
-const AddPersonalBankAccountPage = props => (
-    <ScreenWrapper>
-        <HeaderWithCloseButton
-            title={props.translate('bankAccount.addBankAccount')}
-            onCloseButtonPress={Navigation.dismissModal}
-        />
-        <AddPlaidBankAccount
-            onSubmit={({account, password, plaidLinkToken}) => {
-                BankAccounts.addPersonalBankAccount(account, password, plaidLinkToken);
-            }}
-            onExitPlaid={Navigation.dismissModal}
-        />
-    </ScreenWrapper>
-);
+const AddPersonalBankAccountPage = (props) => {
+    // If we are coming back from the Plaid OAuth flow
+    const receivedRedirectURI = BankAccounts.getOAuthReceivedRedirectURI();
```

Comment:
NAB, probably could use `BankAccounts.getOAuthReceivedRedirectURI()` directly instead of creating a variable + comment.